Collaborative writing platforms section & table

dhimmel opened the pull request:
Closes https://github.com/greenelab/meta-review/issues/195

dhimmel committed:
Update comparison table with Authorea updates

dhimmel commented on content/02.main-text.md:
```diff
@@ -29,20 +29,20 @@ Existing platforms work well for editing text and are widely used for scholarly
 However, they lack features that are important for open writing, such as managing a collaborative manuscript and attributing precise credit among many authors (Table @tbl:platforms).
 None of these platforms offer the ability to address thematically related changes together and enable multiple authors to iteratively refine proposed changes.
 
-| Feature | Manubot | Authorea + BibTeX | Overleaf v2 + BibTeX | Google Docs + Paperpile | Word Online<sup>1</sup> | Markdown on GitHub |
```
I removed "+ BibTeX" because I think that in the case of authorea it may be inaccurate and in the case of Overleaf it may be assumed. 

It seems like BibTex refers to a program and less so the `.bib` file format. https://tex.stackexchange.com/a/299286 And I am not sure if Authorea uses the BibTeX program to render their citations, but I think it is likely that they do not.

slochower replied:
Agree

dhimmel commented on content/02.main-text.md:
```diff
@@ -29,20 +29,20 @@ Existing platforms work well for editing text and are widely used for scholarly
+| Citation by identifier | Yes | Yes | No | No | No | No |
 | Editing software | Any text editor | Web interface | Web interface | Web interface | Web interface | Any text editor |
-| Document format | Markdown | LaTeX | LaTeX | Proprietary | Proprietary | Markdown |
+| Document format | Markdown | HTML | LaTeX | Proprietary | Proprietary | Markdown |
```
It seems like the primary output format is HTML although the input is a mix of HTML exposed via a WYSIWIG-editor, markdown cells and latex cells.

slochower replied:
Agree

agitter reviewed (commented):
These changes to the table all look good to me.  There are a few other parts of the manuscript we can change as well to keep things consistent:

> Others are LaTeX-based systems for technical writing [6], such as Overleaf and Authorea.

If Authorea is not LaTeX-based, we need to rephrase this.

The Table 1 caption says
> We assessed features on June 15, 2018 using the free version of each platform and updated our assessment on April 3, 2019 to add Overleaf v2 and the features in the bottom three rows.

Should we update the date of the Authorea evaluation as well?  I'm not sure how many of these features may have changed since June 2018.

The `Citation by identifier` section says
>Manubot includes an additional layer of citation processing, currently unique to the system.

I believe that phrasing is still accurate even if Authorea has its own citation by identifier approach, right?

dhimmel committed:
Revisions and udpated intro to alternative platforms

dhimmel committed:
Create "Collaborative writing platforms" section

dhimmel committed:
Update response to reviewers

dhimmel committed:
Merge branch 'master' into authorea

dhimmel commented:
I moved the "Collaborative writing platforms" content to its own section rather than in "Manubot contribution workflow". I also rewrote the start of that paragraph to provide a little more context and detail on existing platforms.

slochower commented on content/02.main-text.md:
```diff
@@ -24,36 +24,40 @@ When text is changed, Manubot automatically updates the manuscript's web page so
+In general, platforms with "what you see is what you get" (WYSIWYG) editors, such as Microsoft Word or Google Docs, require the least technical expertise to use.
+On the flip side, WYSIWIG platforms can be difficult to customize and incorporate into automated computational workflows.
+Traditionally, LaTeX has been used for these needs, since documents are written in plain text and the system is open source and customizable.
+Rendering LaTeX documents requires specialized software, but webapps like [Overleaf](https://www.overleaf.com/) now enable collaborative authoring of LaTeX documents via a webapp.
```
Probably don’t need the second webapp.

slochower commented on content/02.main-text.md:
```diff
@@ -24,36 +24,40 @@ When text is changed, Manubot automatically updates the manuscript's web page so
+On the flip side, WYSIWIG platforms can be difficult to customize and incorporate into automated computational workflows.
+Traditionally, LaTeX has been used for these needs, since documents are written in plain text and the system is open source and customizable.
+Rendering LaTeX documents requires specialized software, but webapps like [Overleaf](https://www.overleaf.com/) now enable collaborative authoring of LaTeX documents via a webapp.
+Nonetheless, LaTeX-based systems are limited in that PDF is the only fully supported output format.
```
I might say “usually” here. For a long time DVI was used as the primary output.

dhimmel replied:
What about adding the following qualification:

> Nonetheless, LaTeX-based systems are limited in that PDF (including PDF predecessors) is the only fully supported output format.

dhimmel replied:
Or perhaps `(or similar)`

slochower replied:
I like similiar. It also covers things like .dvju and to me, it implies static and non-interactive which is where HTML output shines. I think LaTeX to HTML usually goes via PDF, so there is little to no interactivity.

slochower commented on content/02.main-text.md:
```diff
@@ -24,36 +24,40 @@ When text is changed, Manubot automatically updates the manuscript's web page so
+Rendering LaTeX documents requires specialized software, but webapps like [Overleaf](https://www.overleaf.com/) now enable collaborative authoring of LaTeX documents via a webapp.
+Nonetheless, LaTeX-based systems are limited in that PDF is the only fully supported output format.
+Alternatively, [Authorea](https://www.authorea.com/) is a collaborative writing webapp whose primary output format is HTML, but allows authors to write sections in Markdown, a limited subset of LaTeX, or their WYSIWIG HTML editor.
+Collaborative writing tools ideally offer version control, multiple permission levels, or other functionality to support multi-author document editing.
```
I think you can merge this sentence with the following sentence to ease the transition because they are both declarative about other tools, in general.

dhimmel committed:
Rearrange table/paragraph, revisions

agitter commented on content/02.main-text.md:
```diff
@@ -24,36 +24,36 @@ When text is changed, Manubot automatically updates the manuscript's web page so
+There are many existing collaborative writing platforms (Table @tbl:platforms) [@doi:10.1038/514127a].
+In general, platforms with "what you see is what you get" (WYSIWYG) editors, such as Microsoft Word or Google Docs, require the least technical expertise to use.
+On the flip side, WYSIWIG platforms can be difficult to customize and incorporate into automated computational workflows.
+Traditionally, LaTeX has been used for these needs, since documents are written in plain text and the system is open source and customizable.
```
` and customizable` feels redundant to me because `these needs` refers to customize and automate in the prior sentence.

agitter commented on content/02.main-text.md:
```diff
@@ -24,36 +24,36 @@ When text is changed, Manubot automatically updates the manuscript's web page so
+Traditionally, LaTeX has been used for these needs, since documents are written in plain text and the system is open source and customizable.
+Rendering LaTeX documents requires specialized software, but webapps like [Overleaf](https://www.overleaf.com/) now enable collaborative authoring of LaTeX documents.
+Nonetheless, LaTeX-based systems are limited in that PDF (or similar) is the only fully supported output format.
+Alternatively, [Authorea](https://www.authorea.com/) is a collaborative writing webapp whose primary output format is HTML, but allows authors to write sections in Markdown, a limited subset of LaTeX, or their WYSIWIG HTML editor.
```
Maybe split into two sentences:
> primary output format is HTML.  It allows authors

Are we sure it only supports a limited subset of LaTeX?  I haven't checked myself, so I'll assume you tested this.

dhimmel replied:
From https://support.authorea.com/en-us/article/how-do-i-include-a-latex-package-1bjtigr/:

> Authorea features [LaTeXML](http://dlmf.nist.gov/LaTeXML/), the most powerful way to render LaTeX notation on the web. [This](https://github.com/Authorea/LaTeXML/tree/master/lib/LaTeXML/Package) is an up to date list of all latex packages you can use on Authorea.

agitter commented on content/02.main-text.md:
```diff
@@ -63,6 +63,13 @@ Edits made by distinct authors may be attributed to a single author.
 
+Existing platforms work well for editing text and are widely used for scholarly writing.
+However, they often lack features that are important for open collaborative writing, such as versatile version control and multiple permission levels.
+Table @tbl:platforms compares several existing platforms.
```
This sentence seems redundant to me.  Can we cut it and add the table reference elsewhere in this paragraph in a parenthetical if you think we need it?

agitter commented on content/response-to-reviewers.md:
```diff
@@ -173,7 +173,7 @@ Same for inline comments and diff colorization.
 We added `WYSIWYG mode`, `Inline comments`, and `Viewing changes` as new features to the table that compares collaborative writing platforms.
 In addition, we updated the table to use Overleaf v2 instead of Overleaf v1, which was discontinued.
-We discussed these changes in [GH147](https://github.com/greenelab/meta-review/issues/147) and edited the table in [GH188](https://github.com/greenelab/meta-review/pull/188).
+We discussed these changes in [GH147](https://github.com/greenelab/meta-review/issues/147) and edited the table in [GH188](https://github.com/greenelab/meta-review/pull/188) and [GH196](https://github.com/greenelab/meta-review/pull/196).
```
Can you also please list this PR in the preceding reviewer comment about `they lack sufficient features for managing a collaborative editing`?  Your new paragraph better explains the difference between Manubot and other tools and gives existing tools their due credit.

agitter reviewed (changes requested):
These are helpful changes.  I have a few more suggestions.

dhimmel committed:
Merge branch 'master' into authorea

dhimmel committed:
Revisions based on gitter review
